src/runtime_src/tools/scripts/build_xrt_with_docker.sh

```diff
@@ -0,0 +1,47 @@
+#!/bin/bash
+
+# SPDX-License-Identifier: Apache-2.0
+# Copyright (C) 2022 Xilinx, Inc. All rights reserved.
+#
+
+# Must be Ubuntu 18.04 or Ubuntu 20.04
+# This will define the linux distro used inside of the docker container
+BASE_IMAGE=${1:-ubuntu:18.04}
```

Comment:
Comment on line #7 states that this docker is 18.04 or 20.04.   

How would one select one or the other? 

I ask for for this script only seems to support 18.04.

Reply:
@rozumx if invoked like this: `./build_xrt_with_docker.sh ubuntu:20.04` it would use the newer version of ubuntu.

However, based on Soren's earlier request (support dbg or opt), I may need to add real argument parsing.